csi/moac/scripts/download_protos.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+set -e
+
+DIR="$(realpath "$(dirname "$0")")"
+COMMIT=7f5d5a10c22d20d4502775d0639e94903773fdcd
```

Comment:
Could this commit be added to somewhere more obvious, maybe in the root of moac? Could we add it, for example, to the package.json?

Reply:
I could probably pass it as an argument from package.json. Would that be ok?

Reply:
yeah that sounds good!